### ISSUE TITLE
Properly evaluate form factors

### ISSUE THREAD

**Issue by mattwthompson** [mattwthompson/scattering#1]
The approximation of `f = Z` can be used

```
for elem in elements:
    ...
    form_factors[elem.symbol] = elem.atomic_number
```

but accurate form factors should be computed or imported from somewhere.

**Comment by rmatsum836** [comment]
TRAVIS uses the Cromer-Mann coefficients: https://doi.org/10.1063/5.0005078

**Comment by rmatsum836** [comment]
![sq](https://user-images.githubusercontent.com/25011342/123673045-9f101700-d805-11eb-86ac-cfc696c9e8aa.png)

I did a test on a system of water this morning comparing this repo to the structure factor calculation from TRAVIS.  The results of this package (using faber-ziman) match up pretty well with TRAVIS.  I suspect the discrepancy is due to the differences in form factors.

**Comment by rmatsum836** [comment]
Addressed in #48
